docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -276,4 +281,75 @@ overall values for the data:
     count    0.222222
     dtype: float64
 
-In every case, the *largest* difference and *smallest* ratio are returned.
\ No newline at end of file
+In every case, the *largest* difference and *smallest* ratio are returned.
+
+
+.. _assessment_predefined_fairness_metrics:
+
+Predefined fairness metrics
+---------------------------
+
+In addition to the disaggregated analysis of base metrics enabled by
+:class:`MetricFrame`, Fairlearn also provides a set of predefined fairness 
+metrics that output a single score. These metrics take as input 
+`sensitive_features` to compute the maximum difference or ratio between 
+subgroups of a sensitive variable. The predefined fairness metrics offered 
+by Fairlearn are **demographic parity** ratio/difference and **equalized odds** 
+ratio/difference. The ratio and difference can be calculated `between_groups` 
+or `to_overall`, but `to_overall` results in more than 1 value being returned.
+:class:`MetricFrame` can also calculate differences and ratios between groups 
+using these methods. For more information on available method of computing 
+ratios or differences, view the documentation for :meth:`MetricFrame.ratio` 
+and :meth:`MetricFrame.difference`, respectively.
+Note that because these metrics are calculated using 
+aggregations between groups, they are meant to be 
+called directly, rather than used within the instantiation of a MetricFrame.
+
+Below, we show an example of calculating demographic parity ratio using the 
+sample data defined above.
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from fairlearn.metrics import demographic_parity_ratio
+    >>> print(demographic_parity_ratio(y_true,
+    ...                                y_pred,
+    ...                                sensitive_features=sf_data))
+    0.66666...
+
+Note that to achieve a one-number score for a standard performance metric,
+like false positive rate or selection rate, the user must use 
```

Comment:
Maybe phrase this as *can* instead of *must*. How about something like:

"It is also possible to define custom fairness metrics based on any standard performance metric (e.g., the false positive rate or AUC) using :func:`make_derived_metric`."

Reply:
Added the suggested sentence!